### ISSUE TITLE
Android on back button pressed

### ISSUE THREAD

**Issue by Jose04Fajardo** [AlejandroRuiz/PayPal.Forms#49]
When i open the PayPal page, if i press the back button the application crash in Android

```
[MonoDroid] UNHANDLED EXCEPTION:
[MonoDroid] System.NullReferenceException: Object reference not set to an instance of an object

[AndroidRuntime] Shutting down VM
[AndroidRuntime] FATAL EXCEPTION: main
[AndroidRuntime] Process: klin.Android, PID: 1532
[AndroidRuntime] android.runtime.JavaProxyThrowable: System.NullReferenceException: Object reference not set to an instance of an object
```

**Comment by AlejandroRuiz** [comment]
Please provide more info like device, OS Version, PayPal.Forms version and also attach a repro demo project

**Comment by AlejandroRuiz** [comment]
Working on a fix new version will be released today

**Comment by AlejandroRuiz** [comment]
Fixed on version 2.18.5

**Comment by Jose04Fajardo** [comment]
thanks
